src/sentry/digests/backends/redis.py

```diff
@@ -128,14 +138,20 @@ def add(self, key, record, increment_delay=None, maximum_delay=None, timestamp=N
             )
         )
 
-    def __schedule_partition(self, host, deadline, timestamp):
-        return script(
+    def __schedule_partition(
+        self, host: int, deadline: float, timestamp: float
+    ) -> Iterable[Tuple[bytes, float]]:
+        # Explicitly typing to satisfy mypy.
```

Comment:
Can't you just type the script function? Also should the return type be `ScriptNode`?